src/locales/pt.json

```diff
@@ -0,0 +1,136 @@
+{
+  "menu": {
+    "superbacked": {
+      "about": "Sobre",
+      "hide": "Ocultar",
+      "quit": "Sair"
+    },
+    "file": {
+      "file": "Arquivo",
+      "create": "Criar",
+      "duplicate": "Duplicar",
+      "restore": "Restaurar",
+      "newWindow": "Nova janela",
+      "closeWindow": "Fechar janela"
+    },
+    "edit": {
+      "edit": "Editar",
+      "undo": "Desfazer",
+      "redo": "Refazer",
+      "cut": "Cortar",
+      "copy": "Copiar",
+      "paste": "Colar",
+      "selectAll": "Selecionar tudo"
+    },
+    "insert": {
+      "insert": "Inserir",
+      "mnemonic": "Mnemônico BIP39",
+      "passphrase": "Senha de recuperação EFF",
+      "scanQrCode": "Escanear código QR"
+    },
+    "view": {
+      "view": "Exibir",
+      "chooseLanguage": "Escolher idioma do aplicativo",
+      "showSelectionAsQrCode": "Mostrar seleção como código QR",
+      "showHiddenSecrets": "Mostrar segredos ocultos",
+      "reload": "Recarregar",
+      "toggleDeveloperTools": "Alternar ferramentas de desenvolvedor"
+    }
+  },
+  "doNot": "Não",
+  "useSuperbackedOnComputer": "use o Superbacked em um computador que não seja",
+  "airGapped": "isolado",
```

Comment:
It is a shame we, in portuguese, don't have the same term as in "air-gapped" with a literal translation. I would use "totalmente isolado" (completely isolated) in this context. Or maybe even better: "isolado (air-gapped)". Let me know what you think in this case, since it is of upmost importance that it is well understood.

Reply:
Hello @irae, 
To be honest, I haven't encountered that expression in Portuguese either, only in English.

I plan to go ahead with a direct translation of @sunknudsen's French version (_"isolé d’Internet"_) and incorporate your suggestion to include the original term in parentheses.

The updated entry will look like this:
```diff
-  "airGapped": "isolado",
+  "airGapped": "isolado da internet (air-gapped)",
```